apps/web/components/sections/project/project-overview.tsx

```
@@ -0,0 +1,66 @@
+        </div>
+
+        <div className="absolute bottom-8 left-8 flex space-x-2">
+          {mediaList.map((item, index) =>
+            index > 0 ? (
+              <img
+                key={index}
+                src={item.src}
+                alt={item.alt}
+                onClick={() => handleThumbnailClick(index)}
+                className="w-24 h-24 rounded object-cover cursor-pointer transition-transform transform hover:scale-105 shadow-sm"
+              />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)